helix-core/src/main/java/org/apache/helix/manager/zk/ZKHelixManager.java

```diff
@@ -292,6 +295,8 @@ public ZKHelixManager(String clusterName, String instanceName, InstanceType inst
         .getSystemPropertyAsInt(SystemPropertyKeys.PARTICIPANT_HEALTH_REPORT_LATENCY,
             ParticipantHealthReportTask.DEFAULT_REPORT_LATENCY);
 
+    _messageRefreshInterval =  HelixUtil.getSystemPropertyAsLong(SystemPropertyKeys.MESSAGE_REFRESH_INTERVAL, VALUE_NOT_SET);
```

Comment:
Do we need a variable for that? I dont think we will reuse the value in later stage.